php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -264,8 +269,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         // check that user has access
         if ($access == false) {
             throw new Exception(
-                "You do not have access to this page.",
-                403
+                "You do not have access to this page."
```

Comment:
The numeric code should not be removed from the exception or it'll break the HTTP return code.

Reply:
This is a generic Exception so I figured this was a mistake because the codes are arbitrary. I don't know of any place in LORIS that translates the int `403` into a proper forbidden response. If these is a place though let me know.

Reply:
main.php and survey.php both do according to `git grep 403`

Reply:
Thanks, hadn't seen that before. I'll add it back.